enterprise/reporting/reporting.go

```diff
@@ -135,7 +138,10 @@ func (r *DefaultReporter) DatabaseSyncer(c types.SyncerConfig) types.ReportingSy
 		MigrationsTable:            "reports_migrations",
 		ShouldForceSetLowerVersion: config.GetBool("SQLMigrator.forceSetLowerVersion", true),
 	}
-	err = m.Migrate("reports")
+	templateData := map[string]int{
+		"AutoVacuumCostLimit": r.autoVacuumCostLimit.Load(),
+	}
+	err = m.MigrateFromTemplates("reports", templateData)
```

Comment:
Question: If we set this in migration, we can change the configuration right, since it will be applied only once?

Reply:
You are right. There is no point int making it a reloadable int